Automattic/wp-calypso
# GF Signup: Experiment social signups with the onboarding-pm flow

Fixes Automattic/growth-foundations#271
Experiment: 21504-explat-experiment
## Proposed Changes

* Loads/assigns the experiment on load of the route `/start/onboarding-pm`
* Checks for variant and redirects to the brand new onboarding flow `/start/onboarding-pm-social`


## Testing Instructions


Case 1
* Assign yourself to treatment
* Go to `/start/onboarding-pm` 
* You should see the Social First passwordless signup page

Case 2
* Assign yourself to treatment
* Go to `/start/onboarding-pm` 
* You should see the legacy signup page

## Pre-merge Checklist
